test/rules/deprecation/test.ts.lint

```diff
@@ -58,9 +61,7 @@ const A = 1, B = 2;
 
 A + B;
 ~     [err % ('A')]
-#if typescript < 2.6.1
     ~ [err % ('B')]
-#endif
```

Comment:
`B` is now marked as deprecated in all TS versions, I didn't mean to change it but I think it's an improvement 😄